WordPress/Classes/ViewRelated/Jetpack/Branding/Overlay/JetpackOverlayViewController.swift

```diff
@@ -0,0 +1,47 @@
+import UIKit
+
+class JetpackOverlayViewController: UIViewController {
+
+    private var redirectAction: (() -> Void)?
+
+    init(redirectAction: (() -> Void)? = nil) {
+        super.init(nibName: nil, bundle: nil)
+        self.redirectAction = redirectAction
+    }
+
+    required init?(coder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+
+    override func loadView() {
+        view = JetpackOverlayView(buttonAction: redirectAction)
+    }
+
+    private func calculatePreferredContentSize() {
```

Comment:
I think we could rename this as `updatePreferredContentSize` or `setPreferredContentSize` as calculate doesn't imply the change on its instance (if my English skills are not failing me here :D )

Reply:
Done!